File: MobyLabWebProgramming.Core/Entities/Product.cs
using MobyLabWebProgramming.Core.Entities;

namespace MobyLabWebProgramming.Backend.Entities;

public class Product : BaseEntity
{
    public string Name { get; set; } = null!;
    
    public string Description { get; set; } = null!;
    
    public long Price { get; set; } 
    
    public string Type { get; set; } = null!;
    
    public string Brand { get; set; } = null!;
    
    public int Stock { get; set; }
    
}

File: src/app/modules/product.ts
export interface Product {
    id: string
    createdAt: string
    updatedAt: string
    name: string
    description: string
    price: number
    type: string
    brand: string
    stock: number
  }

export interface ProductParams{
    search?: string
    page: number
    pageSize: number
  
}
  